.github/workflows/simorgh-integration-tests.yml

```diff
@@ -0,0 +1,30 @@
+name: Simorgh CI - Integration Tests
```

Comment:
Should we keep Simorgh CI in the description?

Reply:
we don't have too but I feel it gives this actions a nice separation from other checks we have running

Reply:
Yes, it looks better when they are namespaced with Simorgh
![image](https://user-images.githubusercontent.com/58214768/97016089-3967de80-1544-11eb-85d9-5b55ca0986d4.png)